changelog/unreleased/add-labels-and-aria-labelled-by-on-login.md

```diff
@@ -0,0 +1,9 @@
+Enhancement: Add labels and aria labelled-by to input elements on login page
+
+https://github.com/owncloud/ocis/pull/1794
+https://github.com/owncloud/ocis/tree/issues/WEB-105
```

Comment:
Branch will be deleted after merge. Could you link to the web github issue instead? Also: why also the empty newlines? ;-)